docs/running-without-web-ui.rst

```diff
@@ -83,4 +83,4 @@ Below is an example that'll set the exit code to non zero if any of the followin
         else:
             environment.process_exit_code = 0
 
-(this code could go into the locustfile.py or in any other file that is imported in the locustfile)
+This code could go into the locustfile.py or in any other file that is imported in the locustfile.
```

Comment:
The parenthesis do actually make a difference. Maybe change it to ”Note that …”

Reply:
I made the change but let me know in case I misunderstood it :)